process/consensus.md

```diff
@@ -59,7 +60,7 @@ official video calls. In the latter case, the consensus vote must be added to
 the agenda at least 24 hours before the video call is scheduled to begin, except
 in the case of general interest votes moving pre-proposals to phase 1, which can
 be added as the discussion proceeds. Consensus decisions are recorded in meeting
-notes and published just like in-person meeting notes are published.
+notes and published just like meeting notes are published.
```

Comment:
looking at this section/paragraph more broadly, this seems to be including full-group VC meetings together with github (and now subgroups), so since we are now basically including VC above, this paragraph should be reworked a little more.

Reply:
Could you give me more specifically on what would be good to rework? In general having all polls in video meetings having a 24-hour notice, and being published in the notes seems like good practice. Or do you mean that you prefer that this is moved to the section above that talks about meetings only?